wafer/users/forms.py

```diff
@@ -37,17 +39,62 @@ class Meta:
 class UserProfileForm(forms.ModelForm):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+        pre_social_index = len(self.fields)
+
+        # currently everything I'm asking for is an url, and this adds some
+        # validation - we may need to revisit this later
+        for field_name in settings.SOCIAL_MEDIA_ENTRIES:
+            self.fields[field_name] = forms.URLField(label=_(settings.SOCIAL_MEDIA_ENTRIES[field_name]), 
```

Comment:
Using `_()` here won't extract the string into our `.po` files. Probably better to wrap it in settings.